colors/onedark.vim

```diff
@@ -285,6 +285,17 @@ call s:h("htmlSpecialTagName", { "fg": s:red })
 call s:h("htmlTag", { "fg": s:white })
 call s:h("htmlTagName", { "fg": s:red })
 
+" Java
```

Comment:
It might be good to see an example screenshot of how the current Java representation works in this theme vs Atom.

I do agree that we don't always want to mirror Atom precisely all the time. A good example of this is the JS highlighting where Atom is a colour overload and this theme is much more sensible.

Reply:
> A good example of this is the JS highlighting where Atom is a colour overload and this theme is much more sensible.

@simonsmith,  I'm curious to see screenshots of what it looks like in your setup.

Here's a side-by-side of mine  (Atom on left, Vim in iTerm2 on right). I'm not sure which I prefer yet, but I can say that Vim's version is pretty red.

![image](https://user-images.githubusercontent.com/794434/40266762-b7ae584c-5b05-11e8-8d88-41dc66c51b64.png)